readme.md

```diff
@@ -29,7 +29,10 @@ weePickle combines some of the best parts of the serialization ecosystem.
 ## sbt
 ```scala
 resolvers += "Rally Health" at "https://dl.bintray.com/rallyhealth/maven"
-libraryDependencies += "com.rallyhealth" %% "weepickle-v1" % "version"
+libraryDependencies ++= Seq(
+  "com.rallyhealth" %% "weepickle-v1" % "version",
+  "com.rallyhealth" %% "weeyaml-v1" % "version"
+)
```

Comment:
Could you move this new dependency info down to the section about YAML?  That's kind of the point of how this is all structured -- it's broken into lots of separate dependencies so that you only need to depend on what you actually need.  So `weeyaml` is only appropriate if you're doing yaml processing.

Reply:
done